README.adoc

```diff
@@ -130,10 +129,9 @@ include::{common-includes}/twyb-end.adoc[]
 
 == Creating a simple application
 
-The simple web application that you will build using Maven and Open Liberty is provided 
-for you in the `start` directory so that you can focus on learning about Maven. The application 
-uses the standard Maven directory structure. Using the standard directory structure saves 
-you from customizing the `pom.xml` file later. 
+The web application that you will use in this guide has been provided for you under 
```

Comment:
I'm not sure why this section has been changed. Can you change it back to how it was?